hummingbird/ml/_parse.py

```diff
@@ -162,33 +171,128 @@ def _parse_sklearn_single_model(scope, model, inputs):
 
 def _parse_sklearn_pipeline(scope, model, inputs):
     """
-    The basic ideas of scikit-learn pipeline parsing:
+    The basic ideas of scikit-learn parsing:
         1. Sequentially go though all stages defined in the considered
            scikit-learn pipeline
-        2. The output `onnxconverter_common.topology.Variable`s of one stage will be fed into its next
+        2. The output variables of one stage will be fed into its next
            stage as the inputs.
-
-    Args:
-        scope: The ``onnxconverter_common.topology.Scope`` for the model
-        model: A `sklearn.pipeline.Pipeline` object
-        inputs: A list of `onnxconverter_common.topology.Variable` objects
-
-    Returns:
-        A list of output `onnxconverter_common.topology.Variable`s produced by the input pipeline
+    :param scope: Scope object defined in _topology.py
+    :param model: scikit-learn pipeline object
+    :param inputs: A list of Variable objects
+    :return: A list of output variables produced by the input pipeline
     """
     for step in model.steps:
         inputs = _parse_sklearn_api(scope, step[1], inputs)
     return inputs
 
 
-def _build_sklearn_api_parsers_map():
-    from sklearn import pipeline
+def _parse_sklearn_feature_union(scope, model, inputs):
```

Comment:
add link to https://github.com/onnx/sklearn-onnx/blob/master/skl2onnx/_parse.py

Reply:
Fixed.

Reply:
Ah but lgbm and xgb are extra now, no? So in theory it should work without all of them (I think).

Reply:
"work" but do nothing